kong/core/reports.lua

```diff
@@ -52,6 +52,22 @@ end
 
 -- UDP logger
 
+local function get_report_value(v)
+  if type(v) == "function" then
+    v = v()
+  end
+
+  if type(v) == "table" then
+    local json, err = cjson.encode(v)
+    if err then
+      log(ERR, "could not JSON encode given table entity: ", err)
+    end
+
+    v = json
+  end
+
+  return v
+end
```

Comment:
style: before merging this, will you update this so we respect the 2 line jumps rule around this function? Thanks!